doc/good_stv_api_0.1-alpha.md

```diff
@@ -0,0 +1,127 @@
+FORMAT: 1A
+
+# Good STV
+
+Good STV is a simple API allowing a group of people to vote on a minor election,
+such as a student society committee position.
+
+# Group Polls
+
+Resources related to polls in the API.
+
+## Poll collection [/polls]
```

Comment:
we Should add a get route to get a list of all current polls

Reply:
I was thinking of this, but wondered if maybe we shouldn't, since we don't want people to be voting for polls they're not in the room for, so why would they need to know this?

Reply:
True if we consider all pollls secret there should be no public get all